src/main/java/uk/gov/ons/ssdc/caseprocessor/messaging/ManagedMessageRecoverer.java

```diff
@@ -79,10 +80,9 @@ public Object recover(RetryContext retryContext) throws Exception {
     logMessage(
         reportResult, retryContext.getLastThrowable().getCause(), messageHash, stackTraceRootCause);
 
-    // Reject the original message where it'll be retried at some future point in time
-    originalMessage.nack();
-
-    return null;
+    // Reject the original message (auto nack'ed). It will be retried at some future point in time
+    throw new MessageHandlingException(
```

Comment:
@NickGrantONS as the ack mode is AUTO (i think) doesn't this mean that the message will be immediately redelivered as opposed to letting the pubsub ack deadline expire and redelivery happen that way? just looking at the docs here: https://cloud.spring.io/spring-cloud-gcp/reference/html/#acknowledging-messages-and-handling-failures

Reply:
it will auto nack it, which will be "immediate" redelivery... i.e. it will redeliver after the redelivery delay we've set on the subscription

Reply:
ok cool, just checking my understanding 👍